lips/lip-2.md

```diff
@@ -0,0 +1,126 @@
+|TreasuryCompliance                                                                                                                             |genesis                     |Globally                                                                                      |0xB1E55ED     |N                   |-                                                                                                                                                                                         |N         |1                                                                                                                                         |
+|Validator                                                                                                                                           |LibraRoot                   |Per Association member                                                                                       |-             |N                   |-                                                                                                                                                                                         |Y         |2                                                                                                                                         |
+|ValidatorOperator                                                                                                                                   |LibraRoot                   |N                                                                                                      |-             |N                   |-                                                                                                                                                                                         |Y         |2                                                                                                                                         |
+|DesignatedDealer                                                                                                                               |LibraRoot                   |N                                                                                                      |-             |Y                   |N                                                                                                                                                                                         |Y         |2                                                                                                                                         |
```

Comment:
According to the current code, DD is granted by TC, which is not consistent to this row.

Reply:
Good catch, will fix.